Wcag2 documents (#1195)
a11yproject/a11yproject.com: src/posts/2020-10-14-wcag2-documents.md

---
layout: post

> title: Introduction to WCAG 2.0

Reviewer: ```suggestion
title: Introduction to WCAG 2.x
```

We'll want to use "2.x" instead of "2.0" to indicate that this post is about versions, 2.0, 2.1, and the current version 2.2, as well as any potential future releases before version 3.0.

> description: A quick overview of WCAG 2 documents.

Reviewer: ```suggestion
description: A quick overview of WCAG 2.x documents.
```
```suggestion
description: A quick overview of WCAG 2 documents.
```

category: Background
author: Hamsa Harcourt
further_reading:

>   - url: https://www.w3.org/TR/WCAG20/#intro-layers-guidance 
>     title:  Web Content Accessibility Guidelines (WCAG) 2.0

Reviewer: ```suggestion
  - url: https://www.w3.org/TR/WCAG22/#intro
    title:  Web Content Accessibility Guidelines (WCAG) 2.2
```

Linking to the most up to date version.

    source: W3C 
  - url:  https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines
    title: Web Content Accessibility Guidelines
    source: Wikipedia
date: 2020-10-14
eleventyNavigation:
  key: {{ category }}
  title: {{ title }}
tags:
  - background
---

> The [**Web Content Accessibility Guidelines**](https://www.w3.org/WAI/standards-guidelines/wcag/) (<abbr>WCAG</abbr>) are web accessibility guidelines published by the **[Web Accessibility Initiative](https://www.w3.org/WAI/about/)** (<abbr>WAI</abbr>). The WCAG's goal is to provide a single shared standard for web content accessibility that meets the needs of individuals, organizations and governments internationally. Over time, WCAG has produced various versions of the [**Web Content Accessibility Guidelines**](https://www.w3.org/WAI/standards-guidelines/wcag/). WCAG 2.0 was published on 11 December 2008. WCAG 2.1 was published on 5 June 2018. WCAG 2.2 is scheduled to be published in 2021.

Reviewer: ```suggestion
The [**Web Content Accessibility Guidelines**](https://www.w3.org/WAI/standards-guidelines/wcag/) (<abbr>WCAG</abbr>) are web accessibility guidelines published by the [**Web Accessibility Initiative**](https://www.w3.org/WAI/about/) (<abbr>WAI</abbr>). 

The WCAG's goal is to provide a single shared standard for web content accessibility that meets the needs of individuals, organizations and governments internationally. Over time, WCAG has produced various versions of the [**Web Content Accessibility Guidelines**](https://www.w3.org/WAI/standards-guidelines/wcag/). WCAG 2.0 was published on 11 December 2008, WCAG 2.1 was published on 5 June 2018. WCAG 2.2 is a W3C working draft as of 5 June 2020.
```

Splitting this up to have a less dense intro. Also updating the bolding application to standardize.

> ## WCAG 2.0

Reviewer: ```suggestion
## WCAG 2.x
```

> WCAG 2.0 consists of 12 guidelines. It is organized around four principles which form the acronym "POUR". They include:

Reviewer: ```suggestion
WCAG 2.x consists of 12 guidelines. It is organized around four principles which form the acronym "POUR". They include:
```

> ## 1. Content must be Perceivable (P).

Reviewer: ```suggestion
### 1. Content must be Perceivable (P).
```

Third level heading, as it's a part of WCAG 2.0

- Provide a text alternative for all non-text content.
- Provide synchronized alternatives for multimedia (such as captioned video, audio descriptions etc).
- Information and structure must be separate from presentation.

> - Make it easy to distinguish foreground information from the background (for example good color contrast).

Reviewer: ```suggestion
- Make it easy to distinguish foreground information from the background (for example: good color contrast).
```

> ## 2. Interfaces must be Operable (O). 

Reviewer: ```suggestion
### 2. Interfaces must be Operable (O). 
```

- All functionality must be operable via keyboard.
- Provide users enough time to read and use content.
- Do not design content in a way that is known to cause seizures.
- Provide mechanisms for users to find content, orientate themselves and navigate through it.

> ## 3. Content and controls must be Understandable (U).

Reviewer: ```suggestion
### 3. Content and controls must be Understandable (U).
```

- Make text content readable and understandable.
- Make the placement and functionality of content predictable.
- Help users avoid and correct mistakes.

> ## 4. Content must be Robust (R).

Reviewer: ```suggestion
### 4. Content must be Robust (R).
```

- Support compatibility with current and future user agents.

## Layers of guidance

> WCAG 2.0 provides various ways to help authors meet its standard called ["layers of guidance"](https://www.w3.org/TR/WCAG20/#intro-layers-guidance). They include the following :

Reviewer: ```suggestion
The WCAG provides various ways to help authors meet its standard called ["layers of guidance"](https://www.w3.org/TR/WCAG20/#intro-layers-guidance). They include the following :
```

- [Guidelines](https://www.w3.org/WAI/WCAG21/quickref/#text-alternatives).
- [Success criteria](https://www.w3.org/WAI/WCAG21/quickref/#audio-only-and-video-only-prerecorded).
- [Techniques](hhttps://www.w3.org/WAI/WCAG21/Understanding/understanding-techniques).
- [Common failures](https://www.w3.org/WAI/WCAG21/Understanding/understanding-techniques#failures).

## Guidelines

> WCAG 2.0 guidelines are categorized into [3 levels of conformance](https://www.w3.org/WAI/WCAG21/Understanding/conformance#levels) in order to meet the needs of different groups and situations. Under each principle, there is a [list of guidelines](https://www.w3.org/TR/WCAG20/#guidelines) that addresses the principles. 

Reviewer: ```suggestion
WCAG 2.x guidelines are categorized into [3 levels of conformance](https://www.w3.org/WAI/WCAG21/Understanding/conformance#levels) in order to meet the needs of different groups and situations. Under each principle, there is a [list of guidelines](https://www.w3.org/TR/WCAG20/#guidelines) that addresses the principles. 
```

## Success criteria

Under each guideline, there are success criteria that describe specifically what is to be done in order to conform to the standard.  Each success criteria is written as a statement that will be either true or false when specific web content is tested against it.  A convenient [list of the success criteria](https://www.w3.org/WAI/WCAG21/quickref/) is also provided.

## Techniques

> The techniques document provides guidance for web authors and evaluators on meeting WCAG 2.0  success criteria. A convenient [list of the WCAG 2.0 techniques](https://www.w3.org/TR/WCAG20-TECHS/) can be found here.

Reviewer: ```suggestion
The techniques document provides guidance for web authors and evaluators on meeting WCAG 2.0 success criteria. A convenient [list of the WCAG 2.0 techniques](https://www.w3.org/TR/WCAG20-TECHS/) can be found here.
```

Remove double space.

## Common failures

> These are common mistakes that are considered failures of Success Criteria of WCAG 2.0. A [comprehensive list of  WCAG 2.0 common failures](https://www.w3.org/TR/WCAG20-TECHS/failures) is also provided.

Reviewer: ```suggestion
These are common mistakes that are considered failures of Success Criteria of WCAG 2.x. A [comprehensive list of  WCAG 2.x common failures](https://www.w3.org/TR/WCAG20-TECHS/failures) is also provided.
```

- People with low vision, and 
- People with disabilities on mobile devices. 

There are additional success criteria in WCAG 2.1 that are not in WCAG 2.0. They are introduced in [What’s New in WCAG 2.1](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-21/). For a detailed  review of WCAG 2.1 , see  [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/).

<blockquote>
	<p>While WCAG 2.0 remains a W3C recommendation, the W3C advises the use of WCAG 2.1 in order to maximize future applicability of accessibility efforts.</p>
	<footer>
		<cite><a href="https://www.w3.org/TR/WCAG21/#abstract/">Web Content Accessibility Guidelines (WCAG) 2.1 - W3C</a></cite>
	</footer>
</blockquote>

In effect, the  publication of WCAG 2.1 does not nullify WCAG 2.0 as content that conforms to WCAG 2.0 also conforms to WCAG 2.1. 

## Summary

The following are important points to remember:

- WCAG is a set of accessibility guidelines published by WAI.
- WCAG 2.0 and WCAG 2.1 are stable, referenceable technical standards.
- They have 12-13 guidelines that are organized under 4 principles: perceivable, operable, understandable, and robust.

> - Content that conforms to WCAG 2.0  is also compatible with  WCAG 2.1.

Reviewer: ```suggestion
- Content that conforms to WCAG 2.0 is also compatible with  WCAG 2.1.
```

Love these summaries you do. Updating to remove a double space.

Author: Thank you

category: Background
author: Hamsa Harcourt
further_reading:
  - url: https://www.w3.org/TR/WCAG22/#intro
    title:  Web Content Accessibility Guidelines (WCAG) 2.2
    source: W3C 
  - url:  https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines
    title: Web Content Accessibility Guidelines
    source: Wikipedia
date: 2020-10-14
eleventyNavigation:
  key: {{ category }}
  title: {{ title }}
tags:
  - background
---

The [**Web Content Accessibility Guidelines**](https://www.w3.org/WAI/standards-guidelines/wcag/) (<abbr>WCAG</abbr>) are web accessibility guidelines published by the [**Web Accessibility Initiative**](https://www.w3.org/WAI/about/) (<abbr>WAI</abbr>). 

The WCAG's goal is to provide a single shared standard for web content accessibility that meets the needs of individuals, organizations and governments internationally. Over time, WCAG has produced various versions of the [**Web Content Accessibility Guidelines**](https://www.w3.org/WAI/standards-guidelines/wcag/). WCAG 2.0 was published on 11 December 2008, WCAG 2.1 was published on 5 June 2018. WCAG 2.2 is a W3C working draft as of 5 June 2020.

## WCAG 2.x

> WCAG 2.x consists of 12 guidelines. It is organized around four principles which form the acronym "POUR". They include:

Reviewer: ```suggestion
WCAG 2.x consists of 12 guidelines. It is organized around four principles which form the acronym "POUR". They include, but are not limited to:
```

### 1. Content must be Perceivable (P).

- Provide a text alternative for all non-text content.
- Provide synchronized alternatives for multimedia (such as captioned video, audio descriptions etc).
- Information and structure must be separate from presentation.
- Make it easy to distinguish foreground information from the background (for example: good color contrast).

### 2. Interfaces must be Operable (O). 

- All functionality must be operable via keyboard.
- Provide users enough time to read and use content.
- Do not design content in a way that is known to cause seizures.
- Provide mechanisms for users to find content, orientate themselves and navigate through it.

### 3. Content and controls must be Understandable (U).

- Make text content readable and understandable.
- Make the placement and functionality of content predictable.
- Help users avoid and correct mistakes.

### 4. Content must be Robust (R).

- Support compatibility with current and future user agents.

## Layers of guidance

> The WCAG provides various ways to help authors meet its standard called ["layers of guidance"](https://www.w3.org/TR/WCAG20/#intro-layers-guidance). They include the following :

Reviewer: ```suggestion
The WCAG provides various ways to help authors meet its standard called ["layers of guidance"](https://www.w3.org/TR/WCAG21/#wcag-2-layers-of-guidance). They include the following :
```

- [Guidelines](https://www.w3.org/WAI/WCAG21/quickref/#text-alternatives).
- [Success criteria](https://www.w3.org/WAI/WCAG21/quickref/#audio-only-and-video-only-prerecorded).
- [Techniques](hhttps://www.w3.org/WAI/WCAG21/Understanding/understanding-techniques).
- [Common failures](https://www.w3.org/WAI/WCAG21/Understanding/understanding-techniques#failures).

## Guidelines

> WCAG 2.x guidelines are categorized into [3 levels of conformance](https://www.w3.org/WAI/WCAG21/Understanding/conformance#levels) in order to meet the needs of different groups and situations. Under each principle, there is a [list of guidelines](https://www.w3.org/TR/WCAG20/#guidelines) that addresses the principles. 

Reviewer: ```suggestion
WCAG 2.x guidelines are categorized into [3 levels of conformance](https://www.w3.org/WAI/WCAG21/Understanding/conformance#levels) in order to meet the needs of different groups and situations. Under each principle, there is a [list of guidelines](https://w3c.github.io/wcag/21/guidelines/) that addresses the principles. 
```

## Success criteria

Under each guideline, there are success criteria that describe specifically what is to be done in order to conform to the standard.  Each success criteria is written as a statement that will be either true or false when specific web content is tested against it.  A convenient [list of the success criteria](https://www.w3.org/WAI/WCAG21/quickref/) is also provided.

## Techniques

> The techniques document provides guidance for web authors and evaluators on meeting WCAG 2.0 success criteria. A convenient [list of the WCAG 2.0 techniques](https://www.w3.org/TR/WCAG20-TECHS/) can be found here.

Reviewer: ```suggestion
The techniques document provides guidance for web authors and evaluators on meeting WCAG 2.1 success criteria. A convenient [list of the WCAG 2.0 techniques](https://www.w3.org/WAI/WCAG21/Techniques/) can be found here.
```

> The techniques document provides guidance for web authors and evaluators on meeting WCAG 2.0 success criteria. A convenient [list of the WCAG 2.0 techniques](https://www.w3.org/TR/WCAG20-TECHS/) can be found here.

Reviewer: ```suggestion
The techniques document provides guidance for web authors and evaluators on meeting WCAG 2.1 success criteria. A convenient [list of the WCAG 2.1 techniques](https://www.w3.org/WAI/WCAG21/Techniques/) can be found here.
```

## Common failures

These are common mistakes that are considered failures of Success Criteria of WCAG 2.x. A [comprehensive list of  WCAG 2.x common failures](https://www.w3.org/TR/WCAG20-TECHS/failures) is also provided.

## What's new?

WCAG 2.1 is the current standard. It was initiated with the aim of improving accessibility guidance for three major groups: 

- People with cognitive or learning disabilities, 
- People with low vision, and 
- People with disabilities on mobile devices. 

> There are additional success criteria in WCAG 2.1 that are not in WCAG 2.0. They are introduced in [What’s New in WCAG 2.1](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-21/). For a detailed  review of WCAG 2.1 , see  [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/).

Reviewer: ```suggestion
There are additional success criteria in WCAG 2.1 that are not in WCAG 2.0. They are introduced in [What’s New in WCAG 2.1](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-21/). For a detailed  review of WCAG 2.1 , see [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/).
```
